lib/word.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'node'
+require 'byebug'
+
+class Word
```

Comment:
This should be renamed to Tree and almost all logic form Node class should be moved here